src/AndroidDebugLauncher/AndroidDebugLauncher.csproj

```diff
@@ -34,6 +34,25 @@
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug.Desktop|AnyCPU'">
```

Comment:
As discussed, I've added a Lab property that can be set on the official build machines rather than adding a bunch of additional build configurations.